src/benchmarks/micro/Serializers/Json_ToString.cs

```diff
@@ -35,5 +44,15 @@ public class Json_ToString<T>
 
         // DataContractJsonSerializer does not provide an API to serialize to string
         // so it's not included here (apples vs apples thing)
+
+        [BenchmarkCategory(Categories.Runtime, Categories.Libraries)]
+        [Benchmark(Description = "SystemTextJson_Reflection")]
+        public string SystemTextJson_Reflection_() => System.Text.Json.JsonSerializer.Serialize(value);
+
+#if NET6_0_OR_GREATER
+        [BenchmarkCategory(Categories.Runtime, Categories.Libraries)]
+        [Benchmark(Description = "SystemTextJson_SourceGen")]
+        public void SystemTextJson_SourceGen_() => System.Text.Json.JsonSerializer.Serialize(value, sourceGenMetadata);
```

Comment:
We need to consume the result. 

```suggestion
        public string SystemTextJson_SourceGen_() => System.Text.Json.JsonSerializer.Serialize(value, sourceGenMetadata);
```